extensions/query-store/src/settings/configureDialog.ts

```diff
@@ -453,53 +508,63 @@ export class ConfigureDialog {
 				fireOnTextChange: true,
 				values: [constants.minuteLabel, constants.hourLabel, constants.dayLabel, constants.automaticLabel],
 				value: constants.automaticLabel,
-				width: cssStyles.configureDialogDropdownWidth
+				width: cssStyles.configureDialogObjectWidth
 			}).component();
 
 		const aggregationSizeLabel = this._view.modelBuilder.text()
 			.withProps({
-				value: constants.aggregationSizeLabel
+				value: constants.aggregationSizeLabel,
+				width: cssStyles.configureDialogObjectWidth
 			}).component();
 
 		const timeIntervalFromRow = this._view.modelBuilder.flexContainer()
-			.withLayout({ flexFlow: 'row' })
-			.withItems([customTimeFromLabel, this.customTimeFromTextBox])
+			.withLayout({ flexFlow: 'row', alignItems: 'baseline' })
+			.withItems([customTimeFromLabel, this.customTimeFromTextBox], { CSSStyles: { flex: '0 0 auto' } })
 			.component();
 
 		const timeIntervalToRow = this._view.modelBuilder.flexContainer()
-			.withLayout({ flexFlow: 'row' })
-			.withItems([customTimeToLabel, this.customTimeToTextBox])
+			.withLayout({ flexFlow: 'row', alignItems: 'baseline' })
+			.withItems([customTimeToLabel, this.customTimeToTextBox], { CSSStyles: { flex: '0 0 auto' } })
 			.component();
 
-		const timeIntervalFromToColumn = this._view.modelBuilder.flexContainer()
+		const timeIntervalComponent = this._view.modelBuilder.flexContainer()
 			.withLayout({ flexFlow: 'column' })
-			.withItems([timeIntervalFromRow, timeIntervalToRow])
+			.withItems([this.timeIntervalOptionsDropdown, timeIntervalFromRow, timeIntervalToRow])
+			.withProps({ ariaRole: 'timeIntervalGroup' })
```

Comment:
I've only seen `ariaRole` set to `radiogroup` for a group of radio buttons and there's a list of well-known values for this property. Should this be `ariaLabel` instead?

Reply:
Updated ariaRole to radiogroup (that was the original intent).